tests/cmsis_rtos_v1/msgq/src/main.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <kernel.h>
+#include <cmsis_os.h>
+
+#define MESSAGE1	512
+#define MESSAGE2	123456
+
+osMessageQDef(message, 5, u32_t);
+osMessageQId message_id;
+
+void send_thread(void const *argument)
+{
+	osStatus status;
+	u32_t data = MESSAGE1;
+
+	/* Prepare and send 1st Message */
+	status = osMessagePut(message_id, data, osWaitForever);
+	zassert_true(status == osOK, "osMessagePut failure for Message1\n");
```

Comment:
In all zasserts, you can remove the \n, it is not needed anymore as it is added by default

Reply:
Yup. Thanks for pointing out.